src/test/java/com/box/sdk/MetadataTemplateTest.java

```diff
@@ -249,4 +250,88 @@ public void testGetEnterpriseMetadataTemplatesParseAllFieldsCorrectly() {
         Assert.assertEquals(secondEntryFieldSecondOption, template.getFields().get(0).getOptions().get(1));
         Assert.assertFalse(iterator.hasNext());
     }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void createMetadataTemplateSucceeds() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        MetadataTemplate.Field ctField = new MetadataTemplate.Field();
+        ctField.setType("string");
+        ctField.setKey("customerTeam");
+        ctField.setDisplayName("Customer Team");
+
+        MetadataTemplate.Field fyField = new MetadataTemplate.Field();
+        fyField.setType("enum");
+        fyField.setKey("fy");
+        fyField.setDisplayName("FY");
+
+        List<String> options = new ArrayList<String>();
+        options.add("FY16");
+        options.add("FY17");
+        fyField.setOptions(options);
+
+        List<MetadataTemplate.Field> fields = new ArrayList<MetadataTemplate.Field>();
+        fields.add(ctField);
+        fields.add(fyField);
+
+        MetadataTemplate template = MetadataTemplate.createMetadataTemplate(api, "enterprise",
```

Comment:
Thanks for the feedback. I catch the error and check the 409 error code. I just added changing the Metadata key as a comment. If we allow Metadata key change every time the test runs, we will have a pile of test Metadata template.